index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>MintPage</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:700" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.2/css/all.css" integrity="sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700" rel="stylesheet">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
+  </head>
+
+  <body>
+    <nav id="navig">
+        <div id="navig-text">
+            <p>CR Trial Training</p>
+            <!-- <p>Already have and Intuit account?</p> -->
+            <a href="#">
+                I forgot my User ID or Password
+            </a>
+        </div>
+        <input id="signin" type="submit" value="Sign In" >
+    </nav>
+    <!--  COMPLETE THE FORM -->
+    <div id="main">
+        <img id="logo" src="images/main-logo.png" alt="logo">
+    </div>
+
+
+    <div class="form-content">
+      <div class="header">One Account. <br/> Everything Intuit.</div>
+      <div class="sub-header">Create an account for all of Intuit's products including Mint. Learn more</div>
+
+      <div class="element">
+        <form action="#" method="post" class="form">
+          <div>
+            <label for="email">Email</label>
+            <input type="email" id="email" class=""/>
+          </div>
+          <div>
+            <label for="phone">Phone (recommanded)</label>
+            <input type="email" id="phone" class=""/>
+            <small>Standard call, messaging or data rate may apply</small>
+          </div>
+          <div>
+            <label for="password">Password</label>
+            <div class="password">
+              <input type="email" id="password" class=""/>
+              <i class="fas fa-unlock"></i>
+            </div>
+          </div>
+          <div>
+            <button class=""><i class="fas fa-lock"></i> Create account</button>
+          </div>
+        </form>
+        <div class="terms">
+          By clicking Create Account, you agree to our
+          License Agreement and have read and acknowledge our Privacy Statement.
+        </div>
+      </div>
+    </div>
```

Comment:
It would be nice to make sure to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) as I said above.
Use the `section` tag instead of the generic `div` one.